### PR TITLE
bpo-37545: 'elif args.verbosity == 1:' in argparse tutorial

### PR DESCRIPTION
**Pull request by aldwinaldwin**
https://bugs.python.org/issue37545

### CODE REVIEW

File: Doc/howto/argparse.rst [hunk 1]
```diff
@@ -646,7 +646,7 @@ to display *more* text instead::
    answer = args.x**args.y
    if args.verbosity >= 2:
        print("Running '{}'".format(__file__))
-   if args.verbosity >= 1:
+   if args.verbosity == 1:
```
Comment (zware) [inline Doc/howto/argparse.rst]: This example must not be changed; it is explicitly about showing how to use "verbosity level to display *more* text"

**Review by zware — CHANGES_REQUESTED** [review]
As noted in the issue I don't think this should be done at all, but the last change especially must not be made as it goes against the point of the example.

File: Doc/howto/argparse.rst (hunk 1 above)
Comment (aldwinaldwin) [reply Doc/howto/argparse.rst]: @zware : ok, now I see the difference in output (filename) compared to previous examples. Agreed.

**Comment by aldwinaldwin** [comment]
I have made the requested changes; please review again

**Pull request closed without merging**